campyros/main.py

```diff
@@ -399,18 +405,17 @@ def fdot(self, time, fn):
         v_relative_wind_b = b2i.inv().apply(v_relative_wind_i)
         air_speed = np.linalg.norm(v_relative_wind_b)
         q = 0.5 * ambient_density * air_speed ** 2  # Dynamic pressure
+        mach = air_speed / speed_of_sound
 
-        if self.parachute_deployed == True and self.parachute.main_c_d != 0:
+        if self.parachute_deployed == True:
             # Parachute forces
-            CD, ref_area = self.parachute.get(alt)
+            CD, ref_area = self.parachute.get(alt,mach)
             F_parachute_i = -0.5 * q * ref_area * CD * v_relative_wind_i / air_speed
-
             # Append to list of forces
             F_i = F_i + F_parachute_i
```

Comment:
Not you who did this but would be good if someone changed anything that is `var=var+var1` to `var+=var1`